.github/workflows/pythonapp.yml

```diff
@@ -150,6 +150,7 @@ jobs:
         # however, "pip install monai*.tar.gz" will build cpp/cuda with an isolated
         # fresh torch installation according to pyproject.toml
         python -m pip install torch>=1.9 torchvision
+        python -m pip install onnx onnxruntime
```

Comment:
@wyli I think we don't need to install here?

Thanks.

Reply:
you are correct @Nic-Ma , I have the line removed - got to know it more.